.github/workflows/wheels.yml

```diff
@@ -46,6 +48,11 @@ jobs:
       id: check
       run: |
         /build/venv/bin/cross-python should_build.py ${{ matrix.package }}
+
+    - shell: bash
+      name: Install compilers
+      run: |
+        apt update && apt install rustc gcc -y
```

Comment:
`gcc` should already be in the container. I don't think pulling `rustc` from apt will work - need the `armv7-unknown-linux-gnueabi` target, and I don't believe Ubuntu has cross target packages for rustc.

Reply:
I realized this afterward, will try and find a way to use ipkg instead of compiling localy, since it would be really tricky to cross compile all this here. thx!